README.md

```diff
@@ -72,12 +78,43 @@ followed by:
 $ heroku rake db:migrate
 ```
 
-To see updated (or data in new fields) data on heroku, you need to run:
+To see updated (or data in new fields) data on heroku, you need to run all of the import tasks:
 
 ```bash
-$ heroku rake import:organisation[{department-slug}]
+$ heroku rake import:all_organisations
+$ heroku rake import:content_items_by_organisation[{department-slug}]
+$ heroku rake import:number_of_views_by_organisation[{department-slug}]
 ```
 
+### Set up Google Analytics credentials in development
+
+To be able to import data from Google Analytics, a few environment variables need to be set.
+
+1) Create files `config/local_env.yml`
+
+N.B. This file has been added to `.gitignore`, so you don't need to worry about accidentally checking it in.
+
+`local_env.yml` is used to 
+* Create the environment variables required to connect to and query the Google API for Google Analytics 
+* Store your Google API credentials. They can either be downloaded from the [google developers console](https://console.developers.google.com/apis/credentials) or from another team member.
+
+2) Add an entry to `config/local_env.yml` for each of the following:
+
+* `CPM_GOVUK_VIEW_ID`
```

Comment:
@pmanrubia I can rename `CPM_GOVUK_VIEW_ID` because this is an environment variable that we have created, but `GOOGLE_PRIVATE_KEY` and `GOOGLE_CLIENT_EMAIL` are used by the Google Client, and we have no control over that code.